src/main/fc/fc_msp.c

```diff
@@ -1478,6 +1481,7 @@ static mspResult_e mspFcProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
         motorConfig()->motorPwmProtocol = constrain(sbufReadU8(src), 0, PWM_TYPE_BRUSHED);
 #endif
         motorConfig()->motorPwmRate = sbufReadU16(src);
+        motorConfig()->digitalIdleOffsetPercent = sbufReadU16(src) / 100.0f;
```

Comment:
This should be length checked, to avoid breaking for users of an old version of the configurator, or some third party tool.